shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -684,6 +684,17 @@ void updateSemantics(ByteBuffer buffer, String[] strings) {
                     event.setScrollX((int) position);
                     event.setMaxScrollX((int) max);
                 }
+                event.setFromIndex(object.scrollIndex);
+                event.setItemCount(object.scrollChildren);
+                if (object.childrenInTraversalOrder != null) {
+                    int visibleChildren = 0;
+                    // There may be invisible children at the beginning and end of the list.
+                    for (SemanticsObject child : object.childrenInTraversalOrder) {
+                        if (!child.hasFlag(Flag.IS_HIDDEN))
+                            visibleChildren += 1;
```

Comment:
Does this produce good results? What if one of the children that's currently visible and included in scrollChildren doesn't contribute semantics? Wouldn't the numbers be misaligned in that case?